kazoo/tests/test_client.py

```diff
@@ -1147,3 +1148,57 @@ def test_unchroot(self):
         client.chroot = '/a'
         self.assertEquals(client.unchroot('/a/b'), '/b')
         self.assertEquals(client.unchroot('/b/c'), '/b/c')
+
+
+class TestReconfig(KazooTestCase):
+
+    def setUp(self):
+        KazooTestCase.setUp(self)
+        if TRAVIS_ZK_VERSION:
+            version = TRAVIS_ZK_VERSION
+        else:
+            version = self.client.server_version()
+        if not version or version < (3, 5):
```

Comment:
Should this kind of check be denoted in the reconfig docstring? (or should that function even do that check itself?)